_data/s23deepak/s23deepak.md

```diff
@@ -0,0 +1,7 @@
+---
+name : Deepak Swaminathan
+institution : Gandhi Institute of Technology and Management, Visakhapatnam

```

Comment:
The institution cannot be longer than 58 characters (you currently have 60 characters).